backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoice/ManualInvoiceRepository.java

```diff
@@ -168,8 +168,11 @@ private I_C_Invoice saveInvoiceHeader(@NonNull final CreateManualInvoiceRequest
 		final int inputDataSourceRecordId = inputDataSourceDAO.retrieveInputDataSource(Env.getCtx(), API_DATA_SOURCE, /* throwEx */true, ITrx.TRXNAME_None)
 				.getAD_InputDataSource_ID();
 
-		invoiceRecord.setAD_Org_ID(createManualInvoiceRequest.getOrgId().getRepoId());
-		invoiceRecord.setC_BPartner_ID(createManualInvoiceRequest.getBillBPartnerId().getRepoId());
+		final OrgId orgId = createManualInvoiceRequest.getOrgId();
+		final BPartnerId billBPartnerId = createManualInvoiceRequest.getBillBPartnerId();
+		final SOTrx soTrx = createManualInvoiceRequest.getSoTrx();
+		invoiceRecord.setAD_Org_ID(orgId.getRepoId());
+		invoiceRecord.setC_BPartner_ID(billBPartnerId.getRepoId());
```

Comment:
Is there any real change between these lines?

Reply:
Nope, reverting